src/components/header.tsx

```diff
@@ -1,42 +1,33 @@
 import { Link } from "gatsby"
-import PropTypes from "prop-types"
 import React from "react"
+import { Box, Text } from "rebass"
 
-const Header = ({ siteTitle }) => (
-  <header
-    style={{
-      background: `rebeccapurple`,
-      marginBottom: `1.45rem`,
-    }}
-  >
-    <div
+const Header: React.FunctionComponent<{ siteTitle: string }> = ({
+  siteTitle,
+}) => (
+  <Box bg="rebeccapurple" mb="1.45rem">
```

Comment:
Props that expect colors will look in your `theme.colors` first, then fallback to treating your string as a color value itself.